x-pack/legacy/plugins/siem/public/components/ml/translations.ts

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const SHOWING = i18n.translate('xpack.siem.anomaliesTable.showing', {
```

Comment:
Missing `type` suffix here from [i18n guidelines](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md#defining-type-for-message).